gcloud/pubsub/topic.py

```diff
@@ -39,9 +42,14 @@ class Topic(object):
     :type connection: :class:gcloud.pubsub.connection.Connection
     :param connection: the connection to use.  If not passed,
                         falls back to the default inferred from the
-                        environment.
+
+    :type add_timestamp_to_messages: boolean
+    :param add_timestamp_to_messages: If true, the topic will add a key,
+                        ``timestamp``, to the attributes of each published
```

Comment:
Length of the argument name made that indentation impractical.  I have verified that this indentation does not break the rendering to HTML (via `tox -e docs && firefox docs/_build/html/pubsub-topic.html`).